lib/galaxy/datatypes/converters/tabular_to_dbnsfp.xml

```diff
@@ -1,9 +1,13 @@
-<tool id="tabular_to_dbnsfp" name="Convert tabular to dbnsfp" version="1.0.1" profile="16.04">
+<tool id="tabular_to_dbnsfp" name="Convert tabular to dbnsfp" version="1.0.2" profile="16.04">
     <description></description>
     <requirements>
-        <requirement type="package" version="0.15.4">pysam</requirement>
+        <requirement type="package" version="1.14">htslib</requirement>
     </requirements>
-    <command>python '$__tool_directory__/tabular_to_dbnsfp.py' '$input' '$dbnsfp.extra_files_path/dbNSFP.gz'</command>
+    <command><![CDATA[
+        mkdir '$dbnsfp.extra_files_path' &&
+        bgzip -c '$input' > '$dbnsfp.extra_files_path/dbNSFP.gz' &&
+        tabix '$dbnsfp.extra_files_path/dbNSFP.gz'
```

Comment:
```suggestion
        tabix -s 1 -b 2 -e 2 '$dbnsfp.extra_files_path/dbNSFP.gz'
```
As in the python form the default values were 0 1 1 and the command line is 1-based while the python is 0-based.